keras/preprocessing/image.py

```diff
@@ -358,6 +358,17 @@ def img_to_array(img, data_format=None):
     return x
 
 
+def save_img(path, x):
+    """Save an image stored as a Numpy Array to a path or file object.
```

Comment:
As I said, using img_to_array would be nicer. The API of `save_img` should reflect the one of img_to_array

Reply:
I forgot to add the change to my last commit - sorry about that.  The `save_img` now reflects the `array_to_img` API.